plugins/callback/foreman.py

```diff
@@ -263,6 +267,7 @@ def append_result(self, result, failed=False):
         value = result._result
         value['failed'] = failed
         self.items[host].append((name, value))
+        self.check_mode = True if result._task.check_mode else False
```

Comment:
isn't that the same as
```suggestion
        self.check_mode = result._task.check_mode
```

Or can `task.check_mode` be anything else? (I don't know)

Reply:
In that case `bool(result._task.check_mode)` should do the trick.

Reply:
You are definitely right, I fixed it.